.github/workflows/image-build-and-draft-release.yaml

```diff
@@ -72,25 +79,25 @@ jobs:
       ## Set docker metatdata
       ## - depending on the matrix.dist, different tags will be enabled
       ## ex. debian will have this tag: `type=ref,event=tag,enable=${{ matrix.dist == 'debian' }}`
-      - name: Docker Metadata ( ${{ matrix.dist }} )
+      - name: Docker Metadata ( ${{ matrix.dist }} ${{ matrix.docker_platforms }} )
         id: docker_metadata
         uses: stacks-sbtc/actions/docker/metadata-action@181f8c67da2707c66b5e31f24e7418c47adefdd1
         with:
           images: |
             ${{ env.ghcr_org }}/sbtc
           tags: |
-            type=raw,value=${{ matrix.docker_target }}-${{ github.event.client_payload.tag_name }}-${{ matrix.dist }}
-            type=raw,value=${{ matrix.docker_target }}-${{ github.event.client_payload.tag_name }},enable=${{ matrix.dist == 'debian' }}
-            type=raw,value=${{ matrix.docker_target }}-latest,enable=${{ env.latest_release == github.event.client_payload.tag_name && matrix.dist == 'debian' }}
-            type=raw,value=${{ matrix.docker_target }}-latest-${{ matrix.dist }},enable=${{ env.latest_release == github.event.client_payload.tag_name }}
+            type=raw,value=${{ matrix.docker_platforms }}.${{ matrix.docker_target }}-${{ github.event.client_payload.tag_name }}-${{ matrix.dist }}
```

Comment:
This creates multiple single-platform images (each with a different tag with the platform), why not creating one multi-platform image instead?